sdk/spring/spring-cloud-azure-starter-monitor/README.md

```diff
@@ -1,6 +1,8 @@
-# Azure Monitor Spring native library for Java
+# Azure Monitor OpenTelemetry Distro / Application Insights in Spring native Java application
```

Comment:
wdyt?

cc @mattmccleary

(ok to rename later too)

```suggestion
# Azure Monitor OpenTelemetry Distro for Spring native
```

Reply:
@trask I have added this topic to a meeting agenda